tasks/FileUploadTask.md

```diff
@@ -17,6 +17,8 @@ This task aims to provide a fluent and easy to use mechanism for the consumer to
 - Using the RequestContext, the feature flag for the **FileUploadTask** can be set for telemetry purposes.
 - The task should not retry to upload failed slices as any retry should already be done by the retry-handler which the task should be using.
 - The task should provide cancellation capabilities through native task cancellation sources when using async APIs.
+- For Outlook, OneDrive and PrintDocument APIs, send a delete request to the upload session URL to cancel an upload task. 
+- The task should provide api to access the upload session object containing its URL, expiry date and the cancellation status.
```

Comment:
I think it'd be interesting to provide a detailed list (and types) of such properties